pkg/sfu/streamtrackermanager.go

```diff
@@ -15,15 +16,14 @@ type StreamTrackerManager struct {
 	trackInfo         *livekit.TrackInfo
 	isSVC             bool
 	maxPublishedLayer int32
+	clockRate         uint32
 
-	configVideo               []StreamTrackerParams
-	configScreenshare         []StreamTrackerParams
-	exemptedLayersVideo       []int32
-	exemptedLayersScreenshare []int32
+	trackerType   config.StreamTrackerType
+	trackerConfig config.StreamTrackerConfig
```

Comment:
Reworking config made this simpler to just take `Video` or `Screenshare` based on track source and put it in this variable and use it.